source/a_box/tmu/tmu.c

```diff
@@ -25,10 +27,12 @@ bool initTMU(tmu_handle_t *tmu) {
    num_bad2 = 0;
    num_bad3 = 0;
    num_bad4 = 0;
-   PHAL_writeGPIO(MUX_1_OUT_Port, MUX_1_OUT_Pin, 0);
-   PHAL_writeGPIO(MUX_2_OUT_Port, MUX_2_OUT_Pin, 0);
-   PHAL_writeGPIO(MUX_3_OUT_Port, MUX_3_OUT_Pin, 0);
-   PHAL_writeGPIO(MUX_4_OUT_Port, MUX_4_OUT_Pin, 0);
+   tmu_daq_override = false;
+   tmu_daq_therm = 0;
+   PHAL_writeGPIO(MUX_A_Port, MUX_A_Pin, 0);
+   PHAL_writeGPIO(MUX_B_Port, MUX_B_Pin, 0);
+   PHAL_writeGPIO(MUX_C_Port, MUX_C_Pin, 0);
+   PHAL_writeGPIO(MUX_D_Port, MUX_D_Pin, 0);
    return false;
```

Comment:
initTMU currently returns false, but in main you removed the !, so it is just going to hardfault. On the same note, we don't really need to make this return a boolean, because it isn't really checking for anything.

Reply:
if (false){ Hardfault}